man/udevadm.xml

```diff
@@ -769,6 +769,14 @@
             <xi:include href="version-info.xml" xpointer="v258"/>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>--revert</option></term>
+          <listitem>
+            <para>Revert previously set configurations.</para>
```

Comment:
This description should be expanded to say exactly what kind of configuration is reverted (ie dynamic one)